### ISSUE TITLE
module 'coloredlogs' has no attribute 'CAN_USE_BOLD_FONT'

### ISSUE THREAD

**Issue by tankerkiller125** [grapheneX/grapheneX#118]
**Describe the bug**
When using SSH to get to a server I'm getting the following error after a fresh install of graphenex

**Log Messages**
```
Traceback (most recent call last):
  File "/usr/local/bin/grapheneX", line 7, in <module>
    from graphenex.__main__ import main
  File "/usr/local/lib/python3.6/dist-packages/graphenex/__main__.py", line 4, in <module>
    from graphenex.core.utils.helpers import parse_cli_args, print_header
  File "/usr/local/lib/python3.6/dist-packages/graphenex/core/utils/helpers.py", line 17, in <module>
    logger = GraphenexLogger(__name__)
  File "/usr/local/lib/python3.6/dist-packages/graphenex/core/utils/logcl.py", line 51, in __init__
    levelname=dict(color='cyan', bold=coloredlogs.CAN_USE_BOLD_FONT),
AttributeError: module 'coloredlogs' has no attribute 'CAN_USE_BOLD_FONT'
```

**Expected behavior**
This shouldn't happen

**Desktop (please complete the following information):**
 - OS: [e.g. Linux / Windows] Ubuntu Server 18.04
 - Browser [e.g. chrome, firefox] N/A
 - Version [e.g. 22] PIP Install (can't find version)

**Additional context**
I have no idea if this would work with an actual monitor as I don't have a physical connection to this server.

**Comment by PiterPentester** [comment]
Hey! 
I have the same problem on CentOS7.
I think the problem is in the python version.
Because I have python3.16 on my VM and the Python package distributed with Ubuntu 18.04 is version 3.6.8.
So after upgrading python to version 3.8 (we need version >= 3.7 ), grapheneX working as expected.

Hope this is helpful!

Regards,
Bohdan

**Comment by jvirgovic** [comment]
@orhun, I believe we can close this issue, because it's not really a gX bug. Any app will most likely throw errors if you try to run it with very old version of the language + if it was caused by some old dependency it was likely already fixed in #137.
